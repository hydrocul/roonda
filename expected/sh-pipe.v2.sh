#!/bin/sh

#################################################
# roonda_35f667a5ef6b240dccabee366adf62baceeec361.pl:
#################################################
cat <<\END_OF_ROONDA_SOURCE_FILE > $ROONDA_TMP_PATH/roonda_35f667a5ef6b240dccabee366adf62baceeec361.pl
use Encode qw/encode/;
use Data::Dumper;

print encode('utf-8', "Hello\n");
END_OF_ROONDA_SOURCE_FILE
#################################################

echo 'Hey!' | cat | perl $ROONDA_TMP_PATH/roonda_35f667a5ef6b240dccabee366adf62baceeec361.pl
echo abc > "$ROONDA_TMP_PATH"/abc.txt
cat < "$ROONDA_TMP_PATH"/abc.txt | cat
cat < "$ROONDA_TMP_PATH"/abc.txt
cat "$ROONDA_TMP_PATH"/abc.txt | tee "$ROONDA_TMP_PATH"/abc1.txt > "$ROONDA_TMP_PATH"/abc2.txt
cat < "$ROONDA_TMP_PATH"/abc1.txt
echo ABC >> "$ROONDA_TMP_PATH"/abc2.txt
cat < "$ROONDA_TMP_PATH"/abc2.txt
