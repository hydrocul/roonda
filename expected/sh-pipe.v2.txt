Hello
abc
abc
abc
abc
ABC
